src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -1,63 +1,43 @@
 import React, { Component } from "react"
-import { ScrollView } from "react-native"
 import { createPaginationContainer, graphql, RelayPaginationProp } from "react-relay"
 import styled from "styled-components/native"
 
 import { LotsByFollowedArtists_query } from "__generated__/LotsByFollowedArtists_query.graphql"
-import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
-import Spinner from "lib/Components/Spinner"
-import { PAGE_SIZE } from "lib/data/constants"
-import { isCloseToBottom } from "lib/utils/isCloseToBottom"
-import { SectionHeader } from "./SectionHeader"
+import { InfiniteScrollArtworksGridContainer as InfiniteScrollArtworksGrid } from "lib/Components/ArtworkGrids/InfiniteScrollArtworksGrid"
+import { SectionHeader as _SectionHeader } from "./SectionHeader"
 
 const DEFAULT_TITLE = "Lots by Artists You Follow"
 
+const Container = styled.View`
+  padding: 10px;
+`
+
+const SectionHeader = styled(_SectionHeader)`
+  padding-bottom: 10px;
+`
+
 interface Props {
   relay: RelayPaginationProp
   title?: string
   query: LotsByFollowedArtists_query
 }
 
-interface State {
-  fetchingMoreData: boolean
-}
-
-export class LotsByFollowedArtists extends Component<Props, State> {
-  state = {
-    fetchingMoreData: false,
-  }
-
-  loadMore = () => {
-    if (!this.props.relay.hasMore() || this.props.relay.isLoading()) {
-      return
-    }
-
-    this.setState({ fetchingMoreData: true })
-    this.props.relay.loadMore(PAGE_SIZE, error => {
-      if (error) {
-        // FIXME: Handle error
-        console.error("LotsByFollowedArtists.tsx", error.message)
-      }
-      this.setState({ fetchingMoreData: false })
-    })
-  }
-
+export class LotsByFollowedArtists extends Component<Props> {
   render() {
-    const artworks = this.props.query.sale_artworks.edges.map(edge => edge.node.artwork)
-    if (artworks.length === 0) {
+    if (this.props.query.me.lotsByFollowedArtistsConnection.edges.length === 0) {
       return null
     }
 
     const { title = DEFAULT_TITLE } = this.props
 
     return (
-      <ScrollView onScroll={isCloseToBottom(this.loadMore)} scrollEventThrottle={400}>
-        <SectionHeader title={title} />
-        <Container>
-          <GenericGrid artworks={artworks as any} />
-          {this.state.fetchingMoreData && <Spinner style={{ marginTop: 20 }} />}
-        </Container>
-      </ScrollView>
+      <Container>
```

Comment:
Nit: May want to wrap in a `<Theme>` and use `<Box p={1}>` here instead of this container. Also the section header can get `pb={1}`

Reply:
I didn’t really want to change much more than what was needed. Would these changes be better in any way other than consistency perhaps?

Reply:
Cleaner, and if the design system ever changes it'll keep it consistent, better practice but not mandatory.

Reply:
Good point re design system changes 👍